fastapi_pagination/ext/cassandra.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, Mapping, Optional, Type, TypeVar
+
+from cassandra.cluster import SimpleStatement
+from cassandra.cqlengine import connection
+from cassandra.cqlengine.models import Model
+
+from ..api import create_page
+from ..bases import AbstractPage
+from ..utils import TParams, verify_params
+
+T = TypeVar("T", bound=Mapping[str, Any])
+
+
+def paginate_with_cursor(
```

Comment:
Let's call it `paginate` as we do it for other integrations.

Reply:
changed